generate.sh

```diff
@@ -63,7 +63,7 @@ function get_realpath() {
 	fi
 }
 
-readonly ASPECTS_DIR="$(dirname "$(get_realpath "${BASH_SOURCE[0]}")")"
+readonly ASPECTS_DIR="$(readlink -f "$(dirname "$(get_realpath "${BASH_SOURCE[0]}")")")"
```

Comment:
`get_realpath` already calls `readlink -f` in way that is supported on both macOS and linux. Is there a particular use case that your change is fixing?

Reply:
```
❯ cat tt.sh      
dirname "$(readlink -f "${BASH_SOURCE[0]}")"
❯ bash tt.sh
/home/user/absolute_path
❯ zsh tt.sh 
.
```

Reply:
No idea why I got a relative path at the time.